spec/contentful/bootstrap/generator_spec.rb

```diff
@@ -3,6 +3,12 @@
 describe Contentful::Bootstrap::Generator do
   subject { Contentful::Bootstrap::Generator.new('wl1z0pal05vy', '48d7db7d4cd9d09df573c251d456f4acc72141b92f36e57f8684b36cf5cfff6e', false) }
 
+  describe 'user agent headers' do
+    it 'client has proper integration data' do
+      expect(subject.client.integration_info).to eq(name: 'bootstrap', version: Contentful::Bootstrap::VERSION)
```

Comment:
Hey @loudmouth,

Yes, the formatting is handled by the SDK. https://github.com/contentful/contentful.rb/blob/master/lib/contentful/client.rb#L241